src/open_mower/config/mower_config.sh.example

```diff
@@ -76,6 +78,9 @@ export OM_UNDOCK_DISTANCE=2.0
 # How many outlines should the mover drive. It's not recommended to set this below 4.
 export OM_OUTLINE_COUNT=4
 
+#Mowing angle -180� - +180�, 0 default angle will be set to the first 2 m of recorded outline, -90° equals north
```

Comment:
- There's a charset issue (you can also write "180 deg" instead)
- Please also specify if its clockwise or counter-clockwise
- I'd personally write "0 = east" instead of "-90 is north" but both is fine for me.